src/Installers/Windows/TargetingPack/TargetingPack.wixproj

```diff
@@ -61,7 +63,7 @@
   <Target Name="ExtractIntermediateTargetingPack" BeforeTargets="PrepareForBuild">
     <PropertyGroup>
       <TargetingPackHarvestRoot Condition="'$(TargetingPackHarvestRoot)' == ''">$(InstallersOutputPath)</TargetingPackHarvestRoot>
-      <IntermediateTargetingPackZip>$(TargetingPackHarvestRoot)aspnetcore-targeting-pack-$(PackageVersion).zip</IntermediateTargetingPackZip>
+      <IntermediateTargetingPackZip>$(TargetingPackHarvestRoot)aspnetcore-targeting-pack-$(TargetingPackVersionPrefix).zip</IntermediateTargetingPackZip>
```

Comment:
@dougbu pretty sure this is it - `TargetingPackVersionPrefix` is `VersionPrefix`, which is `6.0`, whereas PackageVersion is `6.0.0`, so this changes the name of the zip:

https://github.com/dotnet/aspnetcore/blob/8481b938802c769f7c231a7f8d5fd37d3ee90699/eng/Versions.props#L26-L31

@JunTaoLuo was this by design?

Reply:
If it is intentional, we need to change this too:

https://github.com/dotnet/aspnetcore/blob/8481b938802c769f7c231a7f8d5fd37d3ee90699/src/Installers/Windows/TargetingPack/TargetingPack.wixproj#L64

And probably some ingestion stuff in dotnet/installer

Reply:
@wtgodbe are you saying we need to redo the merge to get more of your https://github.com/dotnet/aspnetcore/pull/30603/commits/b8a46a8fe742bc85b9254edbfc63c6ad2a154212 work❔

Reply:
More generally, do we want that in main❔ This might be a no-op merge without it.

Reply:
This change didn't come from mine, it came from https://github.com/dotnet/aspnetcore/commit/69108ceeabe93706622eeacb64caafa2c231a77b. https://github.com/dotnet/aspnetcore/commit/b8a46a8fe742bc85b9254edbfc63c6ad2a154212 is already in main

Reply:
Hmm why did this work in servicing then? Is it because VersionPrefix is 3.1.x in servicing but 6.0 in main? I need to check if there is a better property to use instead of TargetingPackVersionPrefix.

Reply:
Ah I figured it out. This is actually a package version mismatch between TargetingPack.wixproj and App.Ref.csproj.

In the csproj:
```
<VersionPrefix>$(TargetingPackVersionPrefix)</VersionPrefix>
```

but in wixproj it's looking for:
```
<IntermediateTargetingPackZip>$(TargetingPackHarvestRoot)aspnetcore-targeting-pack-$(TargetingPackVersionPrefix).zip</IntermediateTargetingPackZip>
```

Even though TargetingPackVersionPrefix is 6.0.0 in both cases, in the csproj, it's producing the artifact aspnetcore-targeting-pack-6.0.0-ci.zip instead. The `-ci` suffix is not accounted for here. In servicing, there is no such suffix so no errors were encountered in 3.1